docker-compose-smtp.yml

```diff
@@ -0,0 +1,14 @@
+version: '3'
+services:
+  postfix:
+    image: juanluisbaptiste/postfix:latest
+    ports:
+      - "1025:25"
+    environment:
```

Comment:
Nice , an other good option will be to set all smtp credentials in a .env file 
Good job

Reply:
yes it was this way that i meant, 
So awesome !